src/components/Map/MapContainer.js

```diff
@@ -37,7 +38,7 @@ class MapContainer extends React.Component {
     }
     render() {
         return (
-           <Map lat={this.props.lat} lng={this.props.lng} zoom={this.props.zoom} mapStyle={this.props.mapStyle}  UpdateMapStyle={this.props.UpdateMapStyle}
+           <Map lat={this.props.lat} lng={this.props.lng} zoom={this.props.zoom} mapStyle={this.props.mapStyle}  UpdateMapStyle={this.props.UpdateMapStyle} locationList={this.props.locationList}  locationid={locationid}
```

Comment:
Map should only have generic props. Location is a business object, so it shouldn't be tied to Map.

Reply:
If you are trying to render a component inside map, you can pass it as children / named component prop `tooltipContent={<LocationDetails/>}` so that it stays separated